app/audio-player-ui/src/main/kotlin/com/hedvig/android/audio/player/audioplayer/RememberAudioPlayer.kt

```diff
@@ -265,3 +225,18 @@ private class AudioPlayerImpl(
     }
   }
 }
+
+@Composable
+fun rememberAudioPlayer(playableAudioSource: PlayableAudioSource): AudioPlayer {
```

Comment:
This file is now named `RemeberAudioPlayer.kt`.
Maybe we can name it `AudioPlayerImpl.kt` instead since that's the main class in here, while it just also has a `remember` composable function it it?